### PR TITLE
Ugh. Defaults and arrays don't seem to get along.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

**Comment by JakeWharton** [comment]
Can we detect the empty/null array in the run method and use the default path there? Having to specify each time is unfortunate.

**Comment by swankjesse** [comment]
That's a bit closer to what you're requesting, and it also restores backwards-compatibility with the single-valued field. I couldn't figure a way to get the default value without defining a 2nd `@parameter` tag, which is sad. Learning the internals of Mojo might reveal a better solution, but the documentation wasn't helpful for my case.

File: wire-maven-plugin/src/test/resources/exemplar/pom.xml [hunk 1]
```diff
@@ -62,6 +62,9 @@
               <goal>generate-sources</goal>
             </goals>
             <configuration>
+              <protoPaths>
+                <protoPath>${project.basedir}/src/main/proto</protoPath>
+              </protoPaths>
```
Comment (JakeWharton) [inline wire-maven-plugin/src/test/resources/exemplar/pom.xml]: Can be deleted now

**Comment by JakeWharton** [comment]
Lgtm

**Pull request merged**